packages/pytest-simcore/src/pytest_simcore/helpers/faker_webserver.py

```diff
@@ -0,0 +1,55 @@
+from typing import Any, Final
+
+# Web-server API responses of /projects/{project_id}/metadata/ports
+# as reponses in this mock. SEE services/web/server/tests/unit/with_dbs/02/test_projects_ports_handlers.py
+# NOTE: this could be added as examples in the OAS but for the moment we want to avoid overloading openapi.yml
+# in the web-server.
```

Comment:
Question: why would this be an issue?

Reply:
Right now the webserver's OAS is loaded and parsed upon startup on the ``rest`` plugin. This operation is very slow and getting slower with the increasing size of the file. We are slowly replacing this approach.